MediaRecorder.bs

```diff
@@ -410,6 +411,46 @@ interface MediaRecorder : EventTarget {
   </table>
   </dd>
 
+  <dt><dfn method for="MediaRecorder"><code>replaceStream(MediaStream newStream)</code></dfn></dt>
+  <dd>
+  When a {{MediaRecorder}} object&#8217;s {{replaceStream()}} method is invoked,
+  the UA MUST run the following steps:
+  <ol>
+    <li>Let |recorder| be the {{MediaRecorder}} object on which the method was
+    invoked.</li>
+    <li>Let |newStream| be the method's first argument.</li>
+    <li>Let |newTracks| be the set of {{live}} tracks in |newStream|'s
+    [=track set=].</li>
+    <li>Let |stream| be the value of |recorder|'s {{stream}} attribute.</li>
+    <li>Let |tracks| be the set of {{live}} tracks in |stream|'s
+    [=track set=].</li>
+    <li>If the number of audio tracks in |newTracks| is different than the
```

Comment:
@alvestrand That's correct. Upon rejection the application can decide whether to stop the recording or to continue recording with the old track. Would you prefer that failing also triggered stop()?

@guest271314 I don't think the API should silently try to patch the stream so that it will get accepted; what type of tracks you would add e.g. when video is missing is application-specific. Better to inform the application that they're doing something unexpected. Discussing additional APIs to help the application achieve matching streams on the other hand could be discussed separately if you file another issue.